### DISCUSSION TITLE
Builds not auto triggering anymore.

### DISCUSSION THREAD

**Discussion by fatherOfLegends** [codemagic-ci-cd/codemagic-docs#1972] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Auto trigger on PR is not triggering builds. 

Also your docs mention priority support for paying customers, I have been paying for years and this support is not visible to me. Either update your documentation or make the prior support options more visible.

### Expected results

Build should trigger within seconds of pushing to a PR

### Actual results

Builds do not trigger

### Build id (optional)

Not available

**Comment by homaily** [comment]
same here

**Reply by homaily** [reply]
Now the build is triggering but there's a bit for delay

**Comment by CaleyD** [comment]
same - started happening this morning

**Comment by icarusdust** [comment, marked as answer]
Hi @fatherOfLegends, please note that priority support is provided through the chat widget in the UI. GH Discussions channel is for public support. Your team should be able to see a chat widget at the right bottom corner as soon as you log in, please use that widget for quicker support responses. 

As for the issue @fatherOfLegends @homaily @CaleyD , automatic build triggering looks operational atm, please check the Webhook messages for more detailed information under the Webhook tab and let me know if you are still facing the same issue.

**Comment by fatherOfLegends** [comment]
@icarusdust thanks! I did look for the intercom button yesterday but it was not available. It is visible at this point in time. I am not sure what causes it to disappear, next time it is not visible I will capture a screenshot and create a separate issue. Thanks again for the resolution.

**Reply by icarusdust** [reply]
@fatherOfLegends browser extensions/ad blocker could cause it, double check if these are not preventing the widget from appearing. Let me know anyways in case of any questions
